roles/pool/src/main.rs

```diff
@@ -22,68 +27,147 @@ pub type Message = PoolMessages<'static>;
 pub type StdFrame = StandardSv2Frame<Message>;
 pub type EitherFrame = StandardEitherFrame<Message>;
 
-const BLOCK_REWARD: u64 = 5_000_000_000;
-
-//const COINBASE_ADD_SZIE: u32 = 100;
+const BLOCK_REWARD: u64 = 625_000_000;
```

Comment:
Include also this in the issue about automatically update this value

Reply:
After checking spec I realized that we already get the full amount from the TP. Why we need this const? Maybe should be removed.